src/client/game.cpp

```diff
@@ -2696,6 +2687,20 @@ void Game::updatePlayerControl(const CameraOrientation &cam)
 	//tt.stop();
 }
 
+void Game::updatePauseState()
+{
+	bool was_paused = this->m_is_paused;
+	m_is_paused = simple_singleplayer_mode && g_menumgr.pausesGame();
+
+	if (!was_paused && m_is_paused) {
+		this->pauseAnimation();
```

Comment:
Right. I put it in intentionally to be explicit, as I was taught in code review by an experienced developer. If you don't like it, I can remove it.

Reply:
@JosiahWI Explicit `this->` can be nice if it's done consistently. Otherwise, it's usually not worth it, and rather cluttering.

I don't care too much, but if you want to keep it in this function, please also do it for the other instances of `m_is_paused`.